platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/event/TestingEventBuilder.java

```diff
@@ -203,8 +204,8 @@ public TestingEventBuilder(@NonNull final Random random) {
      * @param softwareVersion the software version
      * @return this instance
      */
-    public @NonNull TestingEventBuilder setSoftwareVersion(@Nullable final SoftwareVersion softwareVersion) {
-        this.softwareVersion = softwareVersion;
+    public @NonNull TestingEventBuilder setSoftwareVersion(@Nullable final SemanticVersion softwareVersion) {
+        this.softwareVersion = new BasicSoftwareVersion(softwareVersion.major());
```

Comment:
Maybe for this use case we should introduce a new type that simply wraps the existing PBJ version

Reply:
talked about